src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -93,11 +101,11 @@ public RobotContainer() {
   private void configureBindings() {
 
     new JoystickButton(leftJoystick, 1)
-        .toggleOnTrue(new GripperClose(gripper))
+        .onTrue(new GripperClose(gripper))
         .onFalse(new GripperIdle(gripper));
 
     new JoystickButton(leftJoystick, 2)
-        .toggleOnTrue(new GripperOpen(gripper))
+        .onTrue(new GripperOpen(gripper))
         .onFalse(new GripperIdle(gripper));
```

Comment:
@ParkerMeyers made this change. Not sure the difference between toggleOnTrue vs onTrue.

Reply:
Yes, the gripper controls were not doing anything.  When the electrical team put a meter on the solenoid leads, they saw a short blip when the button was pressed, so something was prematurely turning the gripper solenoids, off.